### PR TITLE
Support context in SSR

### PR DESCRIPTION
**Pull request by galdolber**
First attempt at #66

### CODE REVIEW

File: core/src/uix/core/alpha.cljc [hunk 1]
```diff
@@ -244,7 +244,7 @@
        (into [:> `(.-Provider ~ctx) {:value value}]
              children)
        `(binding [~ctx ~value]
-          ~(into [:<>] children)))))
+          ~(into [:<>! `(fn [f#] (binding [~ctx ~value] (f#)))] children)))))
```
Comment (roman01la) [inline core/src/uix/core/alpha.cljc]: why do we need a new Hiccup form here?

Comment (galdolber) [reply core/src/uix/core/alpha.cljc]: You need to capture the context and bind it when the hiccup is resolved, I don't love this solution but its pretty simple, and only for SSR. #66 explains the issue.

Comment (roman01la) [reply core/src/uix/core/alpha.cljc]: Got it. I'd suggest to come up with a more unique and explicit name, something like `:uix.core.alpha/bind-context`. Also unit tests would be great.

**Comment by galdolber** [comment]
@roman01la check above, I renamed the internal hiccup form to :uix.core.alpha/bind-context and added a test

File: core/src/uix/compiler/alpha.clj [hunk 2]
```diff
@@ -479,6 +479,9 @@
   (when-not (empty? element)
     (let [tag (nth element 0 nil)]
       (cond
+        (identical? :uix.core.alpha/bind-context tag)
```
Comment (roman01la) [inline core/src/uix/compiler/alpha.clj:482]:
```suggestion
        (keyword-identical? :uix.core.alpha/bind-context tag)
```

**Review by roman01la — APPROVED** [review]
Thanks!

File: core/src/uix/compiler/alpha.clj [hunk 3]
```diff
@@ -479,6 +479,9 @@
   (when-not (empty? element)
     (let [tag (nth element 0 nil)]
       (cond
+        (keyword-identical? :uix.core.alpha/bind-context tag)
```
Comment (roman01la) [inline core/src/uix/compiler/alpha.clj]:
```suggestion
        (identical? :uix.core.alpha/bind-context tag)
```

**Pull request merged**